tests/mqtt-limit-msg-1/test.yaml

```diff
@@ -0,0 +1,58 @@
+requires:
+  features:
+    - HAVE_LIBJANSSON
+  files:
+    - rust/src/mqtt/parser.rs
+  script:
+    - grep 'msg-log-limit' suricata.yaml.in > /dev/null
+
+args:
+  - -k none
+
+checks:
+
+  - filter:
+      count: 1
```

Comment:
Why this check ?

Reply:
Ideally, yes, containing: a brief description, Pcap, and Ticket/related issues.

I personally find it useful to add something more descriptive in case people use SV tests to understand how something works, or as examples. It is also where I try to answer questions that might arise about the checks that are made by the test, or the setup used.

Reply:
This first check is just to make sure that nothing in the `connect` log event is changed if log limiting is selected. But technically it is not testing the actual log limiting functionality. I can remove it if it helps with clarity and understandability of the test.

Reply:
OK, will also add a `README.md` for these tests.